src/node/txreconciliation.cpp

```diff
@@ -47,13 +61,18 @@ class TxReconciliationState
     bool m_we_initiate;
 
     /**
-     * TODO: These fields are public to ignore -Wunused-private-field. Make private once used in
-     * the following commits.
-     *
      * These values are used to salt short IDs, which is necessary for transaction reconciliations.
      */
     uint64_t m_k0, m_k1;
 
+    /**
+     * Store all wtxids which we would announce to the peer (policy checks passed, etc.)
```

Comment:
In terms of peers-side policy check (i.e `m_fee_filter_received`), this policy limit is at the tx-relay link level and this is unilaterally initiated by the peer. As such I think there is no guarantee that between time point A we add a `Wtxid` in `m_local_set` and time point B we reconciliate, we have not received a new bip133 message, updating the `m_fee_filter_received`. I believe we can retro-actively stale stored `Wtxid` and as such a bandwidth performance leak, under situations of sudden network mempool spikes.

I don’t think there is that much a tx-announcement strategy (either flooding or reconciliation) can do it in itself, unless assuming some extensions to bip133 messages to commit on a feerate-level duration. As such, I think any improvement is out of scope for this PR.